src/Compilers/Core/Portable/Symbols/NullableEnums.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    // PROTOTYPE(nullable-api): Document
+    public enum NullableAnnotation : byte
+    {
+        Default = 0,
+        Unknown,      // No information. Think oblivious.
+        NotAnnotated, // Type is not annotated - string, int, T (including the case when T is unconstrained).
+        Annotated,    // Type is annotated - string?, T? where T : class; and for int?, T? where T : struct.
+        NotNullable,  // Explicitly set by flow analysis
+        Nullable,     // Explicitly set by flow analysis
```

Comment:
Do we need these two values?

Reply:
Neal is removing them in the master. The next merge will remove them.

---
In reply to: [264906411](https://github.com/dotnet/roslyn/pull/34017#discussion_r264906411) [](ancestors = 264906411)